src/app/service/details/ServiceDetails.jsx

```diff
@@ -1,6 +1,60 @@
-import React from 'react';
+import React, { Component } from 'react';
+import moment from 'moment';
 
-export default function ServiceDetails() {
-  // TODO: Build this page
-  return <div>Service Details</div>;
+import Header from '../../../components/header';
+import Button from '../../../components/button';
+
+import ProgressBar from '../../locationInfo/ProgressBar';
+import FieldItem from '../../locationInfo/FieldItem';
+import NavBar from '../../NavBar';
+
+import { SERVICE_FIELDS } from '../../serviceForm/routes';
+
+// TODO: update fields to have updated value
+const FAKE_UPDATED_AT = moment().subtract(2, 'months');
+
+const getServiceUrl = (locationId, serviceId) => `/location/${locationId}/services/${serviceId}`;
+
+function ServiceHeader({ children }) {
+  return (
+    <div className="container px-4 py-4 text-left">
+      <div className="row">
+        <Header className="m-0">{children}</Header>
+      </div>
+    </div>
+  );
 }
+
+class ServiceDetails extends Component {
+  onGoToDocs = () => {
+    const { locationId, serviceId } = this.props.match.params;
+    this.props.history.push(`${getServiceUrl(locationId, serviceId)}/documents`);
+  };
+
+  render() {
+    const { locationId, serviceId } = this.props.match.params;
+    return (
+      <div className="text-left d-flex flex-column">
+        <NavBar title="Service Details" />
+        <div className="mb-5">
+          <ProgressBar step={1} steps={10} />
+        </div>
+        <ServiceHeader>Check all the Soup Kitchen details</ServiceHeader>
```

Comment:
Since Soup Kitchens aren't special in any way, I assume that part could just be taken from one of the props in the data?

Reply:
Fyi this is updated later